checker/tests/nulless-conservative-defaults/packageannotatedfornullness/Test.java

```diff
@@ -0,0 +1,10 @@
+package packageannotatedfornullness;
```

Comment:
Can you add a `packageannotatedfornullness.notannotated` package that is not `AnnotatedFor` and illustrate what errors you don't expect there?

Reply:
Added, thanks!